### ISSUE TITLE
Incorrect type generated in test files

### ISSUE THREAD

**Issue by roleyfoley** [codeontap/gen3-cli#33]
I've created a testcase file with the following resource property tests
```
{
  "aws_db_postgres_base_db_postgresdbbase_db_postgresdbbase": {
    "filename": "soln-aws-db-postgres-base-template.json",
    "cfn_lint": false,
    "cfn_nag": false,
    "json_structure": {
      "match": [
        {
          "path": "Resources.rdsXdbXpostgresdbbase.Properties.Engine",
          "value": "postgres"
        },
        {
          "path": "Resources.rdsXdbXpostgresdbbase.Properties.EngineVersion",
          "value": "11"
        },
        {
          "path": "Resources.rdsOptionGroupXdbXpostgresdbbaseXpostgres11.Properties.MajorEngineVersion",
          "value": "11"
        }
       ]
}
``` 
Which has the value to test as the string of `"11"` 

However when the test runs it is comparing the template content with a number instead of a string which causes the test to fail 

```
# *********************************************************************
# * TESTCASE aws_db_postgres_base_db_postgresdbbase_db_postgresdbbase *
# *********************************************************************
def test_aws_db_postgres_base_db_postgresdbbase_db_postgresdbbase():
    filename = "soln-aws-db-postgres-base-template.json"

    json_structure = JSONStructure.from_file(filename)

    json_structure.match("Resources.rdsXdbXpostgresdbbase.Properties.Engine", "postgres")
    json_structure.match("Resources.rdsXdbXpostgresdbbase.Properties.EngineVersion", 11)
    json_structure.match("Resources.rdsOptionGroupXdbXpostgresdbbaseXpostgres11.Properties.MajorEngineVersion", 11)
    json_structure.match("Resources.rdsParameterGroupXdbXpostgresdbbaseXpostgres11.Properties.Family", "postgres11")
``` 

The type provided in the testcase should match the type added to the script

**Comment by Neketek** [comment]
Ok, that's clearly a bug.

**Comment by Neketek** [comment]
@roleyfoley Merged #36 Now everything should be fine, please, check it.

**Comment by roleyfoley** [comment]
Tested it and looks like its working as expected. Thanks
